systems/data-plan/base-rate/pkg/utils/utils_test.go

```diff
@@ -35,14 +35,18 @@ func TestRateService_IsFutureDate(t *testing.T) {
 }
 
 func TestRateService_FetchData(t *testing.T) {
-	//Success case
 	mockFileUrl := "https://raw.githubusercontent.com/ukama/ukama/main/systems/data-plan/docs/template/template.csv"
```

Comment:
As discussed last time, as soon as we have a real  url  like "https://raw.githubusercontent.com/ukama/ukama/main/systems/data-plan/docs/template/template.csv" and are making a real network/http call to that url, then this is no more a unit test, but becomes an integration test. 
For now, lets just keep this one here, and I 'll propose to you guys in a couple of days an alternative, and then we will finally move this test (which is a good integration test) into the integration tests pakage.